website/pages/en/querying/querying-the-graph.mdx

```diff
@@ -25,7 +25,19 @@ Each subgraph published to the decentralized Graph Explorer has a unique query U
 
 ![Query Subgraph Pane](/img/query-subgraph-pane.png)
 
-As you can notice, this query URL must use a unique API key. You can create and manage your API keys in the [Subgraph Studio](https://thegraph.com/studio) in the "API Keys" section. Learn more about how to use Subgraph Studio [here](/deploying/subgraph-studio).
+As you can notice, this query URL must use a unique API key. A bearer token can also be used to authenticate the query.
+
+Update the query path:
+```bash
+../api/${api_key}/deployments/... -> ../api/deployments/...
+```
+Add the authorization header:
+```bash
+authorization: Bearer ${api_key}
+```
+Replace ${api_key} with your actual API key.
```

Comment:
~~~suggestion
Update the query path:

```bash
../api/${api_key}/deployments/... -> ../api/deployments/...
```

Add the authorization header:

```bash
authorization: Bearer ${api_key}
```

Replace `${api_key}` with your actual API key.
~~~